sympy/physics/units/systems/cgs.py

```diff
@@ -67,6 +67,8 @@
 cgs_gauss.set_quantity_scale_factor(farad, One/10**5*speed_of_light**2*centimeter)
 cgs_gauss.set_quantity_scale_factor(henry, 10**5/speed_of_light**2/centimeter*second**2)
 
+
+
```

Comment:
we usually recommend max 2 whitelines in unindented code (it's not mandatory, but it makes the code cleaner to read).